modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPreloader.java

```diff
@@ -199,7 +199,11 @@ public boolean disableRebalancingCancellationOptimization() {
 
         AffinityTopologyVersion topVer = top.readyTopologyVersion();
 
-        assert exchFut == null || exchFut.context().events().topologyVersion().equals(top.readyTopologyVersion()) :
+        List<GridDhtPartitionsExchangeFuture> futs = ctx.exchange().exchangeFutures();
```

Comment:
I suggest not to use this method.
The reasons in the previous comment.
Moreover, looks like call result it is never used.

Reply:
Unused variable removed.